app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PagesController < ApplicationController
-  def home; end
-
-  def policy; end
+  before_action :authenticate_user!, except: [:policy]
```

Comment:
`except` is fine, but we do encourage using `only`. So it is more explicit and thus everyone following the same practice.

Also please use symbolic array structure `%i[policy]`